apiserver/common/applicationsetstatus_test.go

```diff
@@ -34,6 +33,14 @@ func (s *serviceStatusSetterSuite) SetUpTest(c *gc.C) {
 	s.leadershipChecker = &fakeLeadershipChecker{isLeader: true}
 }
 
+func (s *serviceStatusSetterSuite) TestStub(c *gc.C) {
+	c.Skip(`This suite is missing tests for the following scenarios:
+- Veryfying that setting application status when identifying as an application fails.
+- Veryfying that setting application status as leader unit works and affects the unit's reported status.
+- Verifying that setting application status as a non-leader fails.
+`)
+}
```

Comment:
This has been completely wiped out by this PR: https://github.com/juju/juju/pull/19225